docs/intermediate-modding/custom-config-syncing.md

```diff
@@ -0,0 +1,233 @@
+
+---
+prev: false
+next: false
+description: An intermediate overview of how to sync custom configs for your Lethal Company mods.
+---
+
+# Custom Config Syncing
+
+::: warning
+**This is an intermediate article. While this introduces some C# concepts, it is recommended to understand C# and the basics of modding this game <i>before</i> reading this article.**
+:::
+
+## Preface
+A very common case for many mod developers is wanting to synchronize the host's configuration file with all other players.
+
+There are many different ways you could achieve this, but we will only go through the two most straight-forward approaches that should work for the majority of cases.
+
+## Automatically Syncing (ConfigurableCompany)
+This is a save-dependent mod that acts as an API, therefore you will need to add it as a dependency if you plan to upload your mod to Thunderstore.
+You can download and learn about it [here](https://thunderstore.io/c/lethal-company/p/AMRV/ConfigurableCompany/) - note that it does not have a **GitHub** repository at the current moment.
+
+With **ConfigurableCompany**, synchronizing configs is quite simple.<br>
+Begin by downloading it and add its DLL as an **Assembly Reference**.
+
+Now we need to bind our config to its own API rather than BepInEx.
+```cs
+ConfigurationBuilder.NewConfig("modder_config_name")
+    .SetName("Configuration name")
+    .SetType(ConfigurationType.Boolean)
+    .SetValue(true)
+    .SetSyncronized(true)
+    .Build();
+```
+
+Alternatively, you can call ```Configuration.Create(identifier, name, category, type, defaultValue)```, though this is not advised due to the complexity of the arguments.
+
+It is also possible set a tooltip or add a category with `.SetTooltip()` and `.NewCategory()` which you can read more about on the [ConfigurableCompany](https://thunderstore.io/c/lethal-company/p/AMRV/ConfigurableCompany/) Thunderstore page.
+
+Now your configs are synced using ConfigurableCompany! See [Synced Config Usage](https://lethal.wiki/intermediate-modding/custom-config-syncing.html#custom-config-syncing) for using synced configs inside your mod.
```

Comment:
Suggest removing this line (or split usage sections into two) as the section this links to is for manual usage and therefore `Instance` will likely not exist.